### PR TITLE
Added logging to shellout method

### PR DESCRIPTION
**Pull request by mcallb**

### CODE REVIEW

**Comment by carpnick** [comment]
:+1:   @gitmlynch - thoughts?  I think this would give us the data we need.

File: lib/mixlibrary/core/shell/scripts/shelloutwrapper.rb [hunk 1]
```diff
@@ -86,11 +86,23 @@ def shellout(eval_error)
...
+                Chef::Log.debug("Command Error:#{result.stderr}")
+                Chef::Log.debug("Command Exit Code:#{result.exitstatus}")
                 return result  
               else
                 result = shellclass.shell("#{@exe} #{@flags} #{@script_file.path}", @shellout_options)
+                Chef::Log.debug("Command output:#{result.stdout}")
+                Chef::Log.debug("Command Error:#{result.stderr}")
+                Chef::Log.debug("Command Exit Code:#{result.exitstatus}")
                 return result
               end
```
Comment (gitmlynch) [inline lib/mixlibrary/core/shell/scripts/shelloutwrapper.rb:107]: Would it make sense to move:

```
Chef::Log.debug("Command output:#{result.stdout}")
Chef::Log.debug("Command Error:#{result.stderr}")
Chef::Log.debug("Command Exit Code:#{result.exitstatus}")
return result
```

to outside of the "if" statement? Those lines appear to be identical in both the "if" and "else" clauses.

**Comment by gitmlynch** [comment]
Other than the failed tests and my comment on the repeated code in the lib/mixlibrary/core/shell/scripts/shelloutwrapper.rb "if" statement, it looks good to me.

**Comment by gitmlynch** [comment]
:+1:

**Pull request merged**